package.json

```diff
@@ -26,19 +26,21 @@
         "@material/form-field": "latest",
         "@polymer/lit-element": "latest",
         "jsdom": "latest",
-        "node-fetch": "latest",
         "reflect-metadata": "latest"
     },
     "devDependencies": {
+        "@types/fetch-mock": "latest",
         "@types/jsdom": "latest",
         "@types/node": "latest",
         "@types/node-fetch": "latest",
         "@types/node-sass": "latest",
         "@types/sinon": "latest",
         "concurrently": "latest",
+        "fetch-mock": "latest",
         "intern": "latest",
         "node-sass": "latest",
         "nodemon": "latest",
+        "node-fetch": "latest",
```

Comment:
@Dom-HMH-MTL I thought we were going to change this for 2.1.2 because of last friday's error?

Reply:
I have updated the signature of the `import` statement from `{ default as fetch }` to `* as fetch`.